dapp/src/components/wizard/pages/ChooseCurrencyType.jsx

```diff
@@ -43,8 +49,35 @@ const CurrencyOption = ({ logo, name, value }) => {
 
 const ChooseCurrencyType = () => {
   const formik = useFormikContext()
+  const account = useSelector(getAccount)
   const communityName = getIn(formik.values, 'communityName')
+  const hasBalance = getIn(formik.values, 'hasBalance')
   const currency = getIn(formik.values, 'currency')
+  const accountAddress = useSelector(state => getAccountAddress(state))
+  const [delay, setDelay] = useState(null)
+  const dispatch = useDispatch()
+  const { jobId, fundingStatus, fundingTxHash } = useSelector(state => state.screens.issuance)
+
+  const handleFund = (e) => {
+    e.preventDefault()
+    dispatch(fund(accountAddress))
+  }
+
+  useEffect(() => {
+    if (jobId) {
+      setDelay(CONFIG.web3.bridge.pollingTimeout)
+    }
+  }, [jobId])
+
+  useEffect(() => {
+    if (fundingStatus) {
```

Comment:
I think you also should check here that the job is done (status is`success ` or `failed`)

Reply:
If it isn’t falsy that means I received the response. In the render, I check his value

Reply:
So you receive a funding object (though it's the job)? Can it be received with a pending status, and then you stop fetchig before the funding is finished?

Reply:
Take a look into `[fetchFundingStatus](https://github.com/fuseio/fuse-studio/blob/master/dapp/src/sagas/user.js#L123)` it updates the `fundingStatus` only if it `success` or `failed`

Reply:
Got it, cool